pkg/discovery/discovery.go

```diff
@@ -148,16 +150,12 @@ func (a *apiDiscovery) getAPIEndpoint(mappings service.ServiceMappings) string {
 	return ""
 }
 
-func (a *apiDiscovery) buildService(ext *ServiceDetail, svc service.Service, p *policy.PolicyItem) (*ServiceDetail, error) {
+func (a *apiDiscovery) buildService(ext *ServiceDetail, svc service.Service, p *policy.PolicyItem) error {
 	isSoap := isSoapAPI(svc.ServiceDetail.Properties.Property)
-	switch isSoap {
-	case true:
+	if isSoap {
```

Comment:
I guess we just care to check the return, don't necessarily need the variable

Reply:
Will remove the variable